apertium-uzb.uzb.lexc

```diff
@@ -90,17 +126,18 @@ Multichar_Symbols
 %<title%>                         !
 
 !! Tense / finite forms
-%<pres%>      ! for "жатыр", "тұр", "отыр" and "жүр"
+%<pres%>      ! for "yotir", "dir",..
 %<aor%>       ! -{E}
-%<past%>      ! -{G}{A}н
+%<past%>      ! -{G}{A}n
 %<ifi%>       ! -{D}{I}
-%<evid%>      ! -{I}пт{I}
+%<evid%>      ! -{I}pt{I}
 %<fut%>       ! -adigan
 %<fut_plan%>  ! -future with -moqchi (planned)
-%<pih%>       ! -{E}тін       ! ~past imperfect ~"used to"
+%<pih%>       ! -{E}tin       ! ~past imperfect ~"used to"
```

Comment:
I believe in Uzbek it's just `-{a}yotgan`, and there's probably a better way to analyse that.

Reply:
Made it -{a}yotgan, at least for now. We can work on analyzing it better way later.